client/src/cl_ctf.cpp

```diff
@@ -378,16 +383,23 @@ void CTF_DrawHud (void)
 
 	if (hasflag)
 	{
-		if (tintglow < 15)
-			tintglowtype = tintglow;
-		else if (tintglow < 30)
-			tintglowtype = 30 - tintglow;
-		else if (tintglow > 45 && tintglow < 60)
-			tintglowtype = tintglow - 45;
-		else if (tintglow >= 60 && tintglow < 75)
-			tintglowtype = 75 - tintglow;
+		if (hud_heldflag_flash == 1)
+		{
+			if (tintglow < 15)
+				tintglowtype = tintglow;
+			else if (tintglow < 30)
+				tintglowtype = 30 - tintglow;
+			else if (tintglow > 45 && tintglow < 60)
+				tintglowtype = tintglow - 45;
+			else if (tintglow >= 60 && tintglow < 75)
+				tintglowtype = 75 - tintglow;
+			else
+				tintglowtype = 0;
+		}
 		else
+		{
 			tintglowtype = 0;
```

Comment:
I think it's better to just initialize tintglowtype = 0 where it's declared at the top and then remove this else